iac-common/src/main/java/org/sonar/iac/common/checks/PropertyUtils.java

```diff
@@ -90,6 +90,10 @@ public static <T extends Tree> Optional<T> get(@Nullable Tree tree, String key,
     return get(tree, key).filter(clazz::isInstance).map(clazz::cast);
   }
 
+  public static <T extends Tree> Optional<T> get(@Nullable Tree tree, Set<String> key, Class<T> clazz) {
```

Comment:
If we're nitpicking, it should probably be `keys` instead of `key` since it's a Set :)